sssom/constants.py

```diff
@@ -177,3 +159,55 @@ class SchemaValidationType(str, Enum):
     SchemaValidationType.JsonSchema,
     SchemaValidationType.PrefixMapCompleteness,
 ]
+
+
+class SSSOMSchemaView:
+    """
+    SchemaView class from linkml which is instantiated when necessary.
+
+    Reason for this: https://github.com/mapping-commons/sssom-py/issues/322
+    Implemented via PR: https://github.com/mapping-commons/sssom-py/pull/323
+    """
+
+    entity_reference = "EntityReference"
+    yaml = pkg_resources.resource_filename("sssom_schema", "schema/sssom_schema.yaml")
+    _view = None
+    _dict = None
+
+    @property
+    def view(self) -> SchemaView:
+        """Return SchemaView object."""
+        if self._view is None:
+            self._view = SchemaView(self.yaml)
```

Comment:
smart, so, whenever self.view is accessed as a variable, this function is executed, effectively deferring the SchemaView() instantiation to the first time its executed? Cool!